client/src/components/Upload.vue

```diff
@@ -113,11 +115,17 @@ export default {
     ImageSequenceType,
   }),
   computed: {
+    ...mapState('Filetypes', ['filetypes']),
     uploadEnabled() {
       return this.location && this.location._modelType === 'folder';
     },
   },
+  created() {
+    this.fetchFiletypes();
+  },
```

Comment:
Both this and `Home.vue` contain this line. I believe `Home.vue` is the only place `Upload.vue` is used, so I think removing it from here and having it solely in `Home.vue` would be best. That way, we're not making unnecessary fetches/store updates, and it's less confusing where/when this action is dispatched.

Reply:
The action contains logic to avoid the API call if the state already exists.

Home.vue does actually need the regexes in the uploaded hook to kick off transcoding.  IMO, that logic should be on the server, but that's an issue for another PR.